### ISSUE TITLE
visual-recognition-v3: swallows daily-transaction-limit-exceeded

### ISSUE THREAD

**Issue by jpbeltrami** [watson-developer-cloud/node-red-node-watson#267]
Spent significant amount of time debugging this issue with no error on the console or in the debug output.

Only found out after doing a ```curl https://gateway-a.watsonplatform.net/visual-recognition/api?api_key=APIKEY```

```
{
    "status": "ERROR",
    "statusInfo": "daily-transaction-limit-exceeded"
}
```

**Comment by chughts** [comment]
I agree visual recognition should not be swallowing errors.

**Comment by chughts** [comment]
Fixed will be released in V 0.5.5

**Comment by chughts** [comment]
V 0.5.5 Released.
